VTEX - Orders API.json

```diff
@@ -797,14 +797,218 @@
+                                                                "type": "integer",
+                                                                "description": "Package invoice value in cents."
+                                                            },
+                                                            "invoiceUrl": {
+                                                                "type": "string",
+                                                                "description": "Package invoice URL.",
+                                                                "nullable": true
+                                                            },
+                                                            "issuanceDate": {
+                                                                "type": "string",
+                                                                "description": "Package invoice issuance date in [ISO 8601 time zone offset format](https://learn.microsoft.com/pt-br/rest/api/storageservices/formatting-datetime-values), as in `YYYY-MM-DD hh:mm:ss[.nnnnnnn]+hh:mm`."
+                                                            },
+                                                            "trackingNumber": {
+                                                                "type": "string",
+                                                                "description": "Package [tracking number](https://developers.vtex.com/docs/api-reference/orders-api#patch-/api/oms/pvt/orders/-orderId-/invoice/-invoiceNumber-).",
+                                                                "nullable": true
+                                                            },
+                                                            "invoiceKey": {
+                                                                "type": "string",
+                                                                "description": "Package invoice key.",
+                                                                "nullable": true
+                                                            },
+                                                            "trackingUrl": {
+                                                                "type": "string",
+                                                                "description": "Package tracking URL.",
+                                                                "nullable": true
+                                                            },
+                                                            "embeddedInvoice": {
+                                                                "type": "string",
+                                                                "description": "Package invoice text in [xml format](https://help.vtex.com/en/tutorial/configuring-xml--tutorials_242). This field is important for stores integrated with external marketplaces."     
```

Comment:
Maybe add a link to external marketplace integration guide?

Reply:
Link added.